src/core_plugins/kibana/server/tutorials/apm/index.js

```diff
@@ -16,7 +16,7 @@ export function apmSpecProvider() {
     artifacts: {
       dashboards: [
         {
-          title: '[APM] Services',
+          id: '[APM] Services',
```

Comment:
This will require updating. I can look up what the ID is.

Reply:
This should be the ID for the apm-server, I tested it by running `apm-server setup`: `8d3ed660-7828-11e7-8c47-65b845b5cfb3`

Reply:
Is it a static value?

Reply:
I'm seeing the same Dashboard ID when running `./apm-server setup` so I assume so.

Reply:
Yes, it should be the similar handled as for beats.